tests/CMakeLists.txt

```diff
@@ -63,7 +61,7 @@ ELSE()
 
   IF("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
     MESSAGE(FATAL_ERROR "The testsuite cannot be configured in-source. "
-      "Please create a separate build directory"
+      "Please create a separate build directory!"
```

Comment:
Did you verify that this actually works? For example, the following on the command line does not do what you want:
```
  echo "abc!abc"
```